<?xml version="1.0"?>
<launch>
  <arg name="scene_file" default="$(find project_ariac)/config/ariac.scene"/>
  <arg name="test" default="false"/>

  <include file="$(find project_ariac)/launch/ur10_moveit.launch"/>
  <node name="conveyor" pkg="project_ariac" type="node"/>

  <node name = "project_ariac_node"
      pkg="project_ariac" type = "project_ariac_node" output = "screen">
      <param name="run" value="$(arg test)"/>
      <param name="end_link" value= "ee_link" />
      <param name="base_link" value= "world" />
      <param name = "elbow_joint" value = "0" />
      <param name = "linear_arm_actuator_joint" value = "3.14" />
      <param name = "shoulder_lift_joint" value = "-1.5" />
      <param name = "shoulder_pan_joint" value = "2.4" />
      <param name = "wrist_1_joint" value = "3.75" />
      <param name = "wrist_2_joint" value = "4.7" />
      <param name = "wrist_3_joint" value = "0.0" />
      <param name = "z_offSet_" value = "0.023" />
      <param name = "planner" value ="RRTConnectkConfigDefault"/>
      <param name = "planning_time" value="400"/>
      <param name = "planning_attempt" value="40"/>
      <param name = "scene" value="$(arg scene_file)"/>
  </node>

</launch>
